NeoMathEngine/src/GPU/CUDA/Kernels/CudaDnnDropoutKernels.h

```diff
@@ -33,4 +34,43 @@ __global__ void ChannelLastBlobSpatialDropoutKernel( const float* __restrict__ i
 	}
 }
 
+__global__ void RandomMatrixDropout( const float* __restrict__ first, int firstHeight,
+	int firstWidth, float* res, int seed, float forwardRate )
+{
+	const unsigned int threshold = forwardRate * UINT_MAX;
+	int row;
+	int col;
+	if (GetCudaTaskIndex2D( firstHeight, ( firstWidth + 3 ) / 4, row, col ) ) {
```

Comment:
_minor_ _codestyle_ 
`if` and `(` must be together.
Spaces inside brackets must be used in the similar approach on both side. But you have `if (GetCuda..` on one side with `col ) )` on the other.